lib/Browser.js

```diff
@@ -165,7 +173,8 @@ Browser.Events = {
   TargetCreated: 'targetcreated',
   TargetDestroyed: 'targetdestroyed',
   TargetChanged: 'targetchanged',
-  Disconnected: 'disconnected'
+  Disconnected: 'disconnected',
+  ConnectionError: 'connectionerror'
```

Comment:
can we issue "disconnected" event for this?

Reply:
Well, my intention was to allow users to handle an error that would otherwise crash the application, and notify them such an error occurred. With `disconnected` you lose that information.